tests/searchControllerTest.js

```diff
@@ -0,0 +1,353 @@
+          expect(res.status).to.equal(201);
+          done();
+        });
+    });
+
+    it('should create a new article', (done) => {
+      const articleDetail3 = {
+        title: 'work yh?',
+        description: 'This is a call for Revolt',
+        body: 'My people the time has come to revolt against this new government',
+        image: 'null'
+      };
+      chai
+        .request(app)
+        .post('/api/articles')
+        .set({ authorization: testToken, Accept: 'application/json' })
+        .send(articleDetail3)
+        .end((err, res) => {
+          expect(res.status).to.equal(201);
+          done();
+        });
+    });
+
+    it('should create a new article', (done) => {
+      const articleDetail4 = {
+        title: 'blah',
+        description: 'This is a call for Revolt',
+        body: 'My people the time has come to revolt against this new government',
+        image: 'null',
+        tagList: ['home', 'react']
+      };
+      chai
+        .request(app)
+        .post('/api/articles')
+        .set({ authorization: testToken, Accept: 'application/json' })
+        .send(articleDetail4)
+        .end((err, res) => {
+          expect(res.status).to.equal(201);
+          done();
+        });
+    });
+
+    it('should create a new article', (done) => {
+      const articleDetail6 = {
+        title: 'home',
+        description: 'This is a call for Revolt',
+        body: 'My people the time has come to revolt against this new government',
+        image: 'null'
+      };
+      chai
+        .request(app)
+        .post('/api/articles')
+        .set({ authorization: testToken, Accept: 'application/json' })
+        .send(articleDetail6)
+        .end((err, res) => {
+          const { slug } = res.body.article;
+          slugged = slug;
+          expect(res.status).to.equal(201);
+          done();
+        });
+    });
+
+    it('should update the article\'s tag', (done) => {
+      const articleTags = {
+        tags: ['angular']
+      };
+      chai
+        .request(app)
+        .post(`/api/articles/${slugged}/tags`)
+        .set({ authorization: testToken, Accept: 'application/json' })
+        .send(articleTags)
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body.message).to.equal('Updated article tags successfully');
+          done();
+        });
+    });
+
+    it('should search by article', (done) => {
+      chai
+        .request(app)
+        .get('/api/articles/search?article=enough&author=false&tag=false')
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body.success).to.equal(true);
+          expect(res.body.articles).to.be.a('array');
+          done();
+        });
+    });
+
+    it('should search by tag', (done) => {
+      chai
+        .request(app)
+        .get('/api/articles/search?article=false&author=false&tag=angular')
+        .end((err, res) => {
+          console.log(res.body);
```

Comment:
Hi. Please, could you remove the console statement here? Thanks

Reply:
Done